examples/layers/preprocessing/random_hue_demo.py

```diff
@@ -16,38 +16,23 @@
 are loaded, then are passed through the preprocessing layers.
 Finally, they are shown using matplotlib.
 """
-import matplotlib.pyplot as plt
-import tensorflow as tf
-import tensorflow_datasets as tfds
+import utils
 
 from keras_cv.layers import preprocessing
 
-IMG_SIZE = (224, 224)
-BATCH_SIZE = 64
-
-
-def resize(image, label):
-    image = tf.image.resize(image, IMG_SIZE)
-    return image, label
-
 
 def main():
-    data, ds_info = tfds.load("oxford_flowers102", with_info=True, as_supervised=True)
-    train_ds = data["train"]
+    # Prepare flower dataset dataset.
+    train_ds = utils.prepare_dataset()
 
-    train_ds = train_ds.map(lambda x, y: resize(x, y)).batch(BATCH_SIZE)
+    # Prepare augmentation layer.
     random_hue = preprocessing.RandomHue(factor=(0.0, 1.0), value_range=(0, 255))
-    train_ds = train_ds.map(
-        lambda x, y: (random_hue(x), y), num_parallel_calls=tf.data.AUTOTUNE
-    )
-
-    for images, labels in train_ds.take(1):
-        plt.figure(figsize=(8, 8))
-        for i in range(9):
-            plt.subplot(3, 3, i + 1)
-            plt.imshow(images[i].numpy().astype("uint8"))
-            plt.axis("off")
-        plt.show()
+
+    # Apply augmentation.
+    train_ds = train_ds.map(lambda x, y: (random_hue(x), y))
+
+    # visualize.
+    utils.visualize_dataset(train_ds)
```

Comment:
@LukeWood #446

Reply:
Missed this PR, thanks Kartik!  I will be sure to do a pass on this!